.github/ISSUE_TEMPLATE/bug.yml

```
@@ -25,6 +25,11 @@ body:
           required: true
         - label: I have resolved all warnings from `brew doctor` and that did not fix my problem.
           required: true
+        - label: I have unset `HOMEBREW_{BOTTLE,ARTIFACT}_DOMAIN` and `HOMEBREW_{BREW,CORE}_GIT_REMOTE` and am still able to reproduce this issue.
+          required: true
+  - type: markdown
+    attributes:
+      value: If this bug is related to mirror sites, please refer to https://github.com/Homebrew/discussions/discussions/1917 for more information, and report bugs to respective mirror maintainers.
```

I'm pretty sure we can put links. Assuming yes, I think this is a bit nicer to read

```suggestion
      value: If this bug is related to mirror sites, please refer to [this disucssion](https://github.com/Homebrew/discussions/discussions/1917) for more information, and report bugs to respective mirror maintainers.
```